test/apigateway.test.ts

```diff
@@ -0,0 +1,118 @@
+import * as cdk from "aws-cdk-lib";
+import { Match, Template } from "aws-cdk-lib/assertions";
+import { User } from "aws-cdk-lib/aws-iam";
+import { AtatRestApi, RestApiVpcConfiguration } from "../lib/constructs/apigateway";
+import * as ec2 from "aws-cdk-lib/aws-ec2";
```

Comment:
```suggestion
import * as cdk from "aws-cdk-lib";
import * as ec2 from "aws-cdk-lib/aws-ec2";
import * as iam from "aws-cdk-lib/aws-iam";
import { Match, Template } from "aws-cdk-lib/assertions";
import { AtatRestApi, RestApiVpcConfiguration } from "../lib/constructs/apigateway";
```

Can we organize imports (here and elsewhere)? Note that this suggestion can't be directly applied. Usage of `User` needs to change to `iam.User`.